### DISCUSSION TITLE
Variables with calc function

### DISCUSSION THREAD

**Discussion by s0la** [davatorium/rofi#1667] [Theming] [answered]
Hi guys!

Just wondering if there is any way I can use variables defined within rasi files as a part of calc function.. I found older request regarding the same subject, and it mentions environment variables, but it's not clear about the way it should be used..

Please let me know if you know anything about this,
Thanks in advance! :)

**Comment by DaveDavenport** [comment, marked as answer]
You cannot use variables in calc function (as stated in the issue).

This might work with env variable:
```bash
MYWIDTH=1000 rofi -theme-str 'window {  width: calc( ${MYWIDTH} - 20 ); }'  -show drun
```

**Reply by s0la** [reply]
Excellent! Thanks for providing an example, I think this will be good enough for my use case!
Thanks again, cheers! :)
